lib/virtio_mmio/virtio_mmio_drv.c

```diff
@@ -286,7 +286,7 @@ struct virtqueue *virtio_mmio_setup_virtqueue(struct virtio_device *vdev,
 
 	vring_info->io = vmdev->shm_io;
 	vring_info->info.num_descs = virtio_mmio_get_max_elem(vdev, idx);
-	vring_info->info.align = VRING_ALIGNMENT;
+	vring_info->info.align = VIRTIO_AMP_VRING_ALIGNMENT;
```

Comment:
The `align` is used by `vq_ring_init`.  That's means that you align the `vring_used` struct on a new 4k page right?
That would mean that the vring is on two 4k page...